local/include/core/ProxyVerifier.h

```diff
@@ -202,148 +212,252 @@ class RuleCheck {
+      std::list<swoc::TextView> localized_values,
+      swoc::TextView rule_type);
 
   /** Generate @a EqualityCheck, invoked by the factory function when the
    * "equals" flag is present.
    *
-   * @param node TextView holding the name of the target field
-   * @param name TextView holding the associated value with the target field,
+   * @param node The name of the target field
+   * @param name The associated value with the target field,
    * that is used with strcasecmp comparisons
    * @return A pointer to the EqualityCheck instance generated, holding key and
    * value TextViews for the rule to compare inputs to
    */
-  static std::shared_ptr<RuleCheck> make_equality(swoc::TextView name,
-                                                  swoc::TextView value);
+  static std::shared_ptr<RuleCheck> make_equality(
+      swoc::TextView name,
+      swoc::TextView value);
+
+  /**
+   * @param values The list of values to expect in the response.
+   */
+  static std::shared_ptr<RuleCheck> make_equality(
+      swoc::TextView name,
+      std::list<swoc::TextView> values);
 
   /** Generate @a PresenceCheck, invoked by the factory function when the
    * "absence" flag is present.
    *
-   * @param name TextView holding the name of the target field
-   * @param value TextView (unused) in order to have the same signature as
+   * @param name The name of the target field
+   * @param value (unused) Used in order to have the same signature as
    * make_equality
    * @return A pointer to the Presence instance generated, holding a name
    * TextView for the rule to compare inputs to
    */
-  static std::shared_ptr<RuleCheck> make_presence(swoc::TextView name,
-                                                  swoc::TextView value);
+  static std::shared_ptr<RuleCheck> make_presence(
+      swoc::TextView name,
+      swoc::TextView value);
+
+  /**
+   * @param values (unusesd) The list of values specified in the YAML node.
+   */
+  static std::shared_ptr<RuleCheck> make_presence(
+      swoc::TextView name,
+      std::list<swoc::TextView> values);
 
   /** Generate @a AbsenceCheck, invoked by the factory function when the
    * "absence" flag is present.
    *
-   * @param name TextView holding the name of the target field
-   * @param value TextView (unused) in order to have the same signature as
+   * @param name The name of the target field
+   * @param value (unused) Used in order to have the same signature as
    * make_equality
    * @return A pointer to the AbsenceCheck instance generated, holding a name
    * TextView for the rule to compare inputs to
    */
-  static std::shared_ptr<RuleCheck> make_absence(swoc::TextView name,
-                                                 swoc::TextView value);
+  static std::shared_ptr<RuleCheck> make_absence(
+      swoc::TextView name,
+      swoc::TextView value);
+
+  /**
+   * @param values (unusesd) The list of values specified in the YAML node.
```

Comment:
"unusesd"? Also, don't pull lists out of YAML nodes as arguments - pass the node.